### PR TITLE
Tag Luxor.jl v0.8.5

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaGraphics/Luxor.jl](https://github.com/JuliaGraphics/Luxor.jl)
Release: [v0.8.5](https://github.com/JuliaGraphics/Luxor.jl/releases/tag/v0.8.5)
Travis: [Travis Build Status](https://travis-ci.org/JuliaGraphics/Luxor.jl/builds/233169497)
Diff: [vs v0.8.4](https://github.com/JuliaGraphics/Luxor.jl/compare/23927e60e1b70875f7d394bbc51056dc145a9a5a...7745d1149a9ca5c86726126f50638769e2fff771)
`requires` vs v0.8.4: 
```diff
--- 0.8.4/requires
+++ 0.8.5/requires
@@ -3,3 +3,4 @@
 Colors
 Cairo
 FileIO
+Juno
```
cc: @cormullion

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

**Comment by tkelman** [comment]
If you directly do `using Hiccup` then Hiccup should be in `REQUIRE`

**Comment by cormullion** [comment]
@tkelman Oops I forgot that. Do I need to do another release?

**Comment by KristofferC** [comment]
You can do the change, delete the current release, create a new one with the same name and attobot will update this PR

File: Luxor/versions/0.8.5/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+7745d1149a9ca5c86726126f50638769e2fff771
```
Comment (tkelman) [inline Luxor/versions/0.8.5/sha1]: should be https://github.com/JuliaGraphics/Luxor.jl/commit/4f23c7a79d4c6d835c4f5b3b1a74452812d9072c ?

Comment (cormullion) [inline Luxor/versions/0.8.5/sha1]: Yes. :(

**Comment by cormullion** [comment]
@tkelman Thanks!

**Pull request merged**